src/routes/recipe.js

```diff
@@ -1,17 +1,62 @@
 import React from 'react';
 import { useParams } from "react-router-dom";
 import { Link } from "react-router-dom";
+import { getRecipe } from '../recipes';
+
+function IngredientItem(props) {
+  const listItems = props.ingredients.map(ingredient => 
+    <li key={ingredient.ingredient_name} className='justify-self-center text-center'>
+      <img src={ingredient.ingredient_img} alt={ingredient.ingredient_name} className='object-cover rounded-full w-20 h-20 md:w-24 md:h-24 mb-2'></img>
+      <div>{ingredient.ingredient_name}</div>
+      <div>{ingredient.ingredient_quantity}</div>
+    </li>);
+  return listItems;
+}
+
+function RecipeSteps(props) {
+  const listItems = props.instructions.map(instructions => <li key={instructions}>{instructions}</li>);
+  return listItems;
```

Comment:
Similar minor comment here:

```suggestion
  return props.instructions.map(instructions => <li key={instructions}>{instructions}</li>);
```